src/query/expression/src/converts/to.rs

```diff
@@ -37,3 +40,38 @@ pub fn to_schema(schema: &TableSchema) -> common_datavalues::DataSchema {
         .collect();
     common_datavalues::DataSchema::new_from(fields, schema.meta().clone())
 }
+
+pub fn scalar_to_datavalue(scalar: &Scalar) -> DataValue {
```

Comment:
@drmingdrmer  this function is re add  because we use DataValue's hash to apply old bloom filter.